README.md

```diff
@@ -24,6 +24,12 @@ See
 <https://github.com/dsaidgovsg/spark-k8s-addons#how-to-properly-manage-pip-packages>
 for more information.
 
+For builds involving Airflow v2 onwards, note that `poetry` is not supported 
+as an installation tool.
+See <https://github.com/apache/airflow/issues/13149> for a related discussion 
+and how to resolve possible conflicts when installing packages on top of this 
+base image.
+
```

Comment:
This set of comment becomes a little weird because we ended up still using `poetry` and still works in practice.

Maybe can rephrase along the line of:

- "We note that it's not officially supported"
- "But we need it anyway to make sure the deps versions are well compatible across multiple installs"
- "And turns out the whole thing still works"

Is this better? Do keep the https://github.com/apache/airflow/issues/13149 link though.

Reply:
Agree that i should elaborate further in this section. Fixed in https://github.com/dsaidgovsg/airflow-pipeline/pull/92/commits/2617a6831a93acb56e900eeee6638938e4380611#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R22-R27